tests/layer_tests/tensorflow_tests/test_tf_SegmentSum.py

```diff
@@ -35,37 +43,76 @@ def create_segment_sum_net(self, data_shape, segment_ids_shape, data_type, segme
-                               use_legacy_frontend):
-        if not use_legacy_frontend:
-            pytest.skip("SegmentSum operation is not supported via legacy frontend.")
-        self._test(*self.create_segment_sum_net(**params),
-                   ie_device, precision, ir_version, temp_dir=temp_dir,
-                   use_legacy_frontend=use_legacy_frontend)
+    @pytest.mark.xfail(
+        condition=platform.system() == "Darwin" and platform.machine() == "arm64",
+        reason="Ticket - 122716",
+    )
+    def test_segment_sum_basic(
+        self, params, ie_device, precision, ir_version, temp_dir, use_legacy_frontend
+    ):
+        self._test(
+            *self.create_segment_sum_net(**params),
+            ie_device,
+            precision,
+            ir_version,
+            temp_dir=temp_dir,
+            use_legacy_frontend=use_legacy_frontend
+        )
 
     test_data_different_types = [
-        dict(data_shape=[2, 3], segment_ids_shape=[2], data_type=tf.int32, segment_ids_type=tf.int32),
-        dict(data_shape=[3, 2], segment_ids_shape=[3], data_type=tf.float64, segment_ids_type=tf.int32),
-        dict(data_shape=[3, 1, 2], segment_ids_shape=[3], data_type=tf.float32, segment_ids_type=tf.int64),
-        dict(data_shape=[4, 2, 1], segment_ids_shape=[4], data_type=tf.float64, segment_ids_type=tf.int64),
+        dict(
+            data=[2, 3],
+            segment_ids=[2],
+            data_type=tf.int32,
+            segment_ids_type=tf.int32,
+        ),
+        dict(
+            data=[3, 2],
+            segment_ids=[3],
+            data_type=tf.float64,
+            segment_ids_type=tf.int32,
+        ),
+        dict(
+            data=[3, 1, 2],
+            segment_ids=[3],
+            data_type=tf.float32,
+            segment_ids_type=tf.int64,
+        ),
+        dict(
+            data=[4, 2, 1],
+            segment_ids=[4],
+            data_type=tf.float64,
+            segment_ids_type=tf.int64,
+        ),
     ]
 
     @pytest.mark.parametrize("params", test_data_different_types)
     @pytest.mark.nightly
-    def test_segment_sum_different_types(self, params, ie_device, precision, ir_version, temp_dir,
-                                         use_legacy_frontend):
-        if not use_legacy_frontend:
-            pytest.skip("SegmentSum operation is not supported via legacy frontend.")
-        self._test(*self.create_segment_sum_net(**params),
-                   ie_device, precision, ir_version, temp_dir=temp_dir,
-                   use_legacy_frontend=use_legacy_frontend)
+    def test_segment_sum_different_types(
+        self, params, ie_device, precision, ir_version, temp_dir, use_legacy_frontend
+    ):
+        self._test(
+            *self.create_segment_sum_net(**params),
+            ie_device,
+            precision,
+            ir_version,
+            temp_dir=temp_dir,
+            use_legacy_frontend=use_legacy_frontend
+        )
```

Comment:
please do not change existing test suite. Just create separate test suite for complex type like in example PR.